src/styles/index.scss

```diff
@@ -1,3 +1,35 @@
+@import './utils/variables';
+@import './blocks/stars';
+@import './blocks/card';
+@import './blocks/header';
+
 body {
   margin: 0;
 }
+
+.container {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, 200px);
+  padding: 50px 40px;
+  gap: 46px 48px;
+  box-sizing: border-box;
+}
+
+@media (min-width: 488px) {
+  .container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    grid-template-columns: repeat(4, 200px);
```

Comment:
The value `200px` for `grid-template-columns` is repeated across different media queries. This is a magic number and should be stored in a variable for better maintainability. You can create a variable like `$column-width: 200px;` and use it across your SCSS.